src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1

```diff
@@ -527,6 +619,10 @@ class SVTResourceResolver: AzSKRoot {
                                 }
                             }
 
+                            if($null -ne $varGroups) {
+                                $projectData["variablegroups"] = $varGroups.Length;
```

Comment:
what if a specific name of var grp was specified in scan command? - that will make length of $varGrps equal to 1.